.github/workflows/wheels.yml

```diff
@@ -2,11 +2,9 @@
 name: Wheel builder
 
 on:
-  push:
-    branches:
-      - master
-      # Release branches
-      - "[0-9]+.[0-9]+.X"
+  create:
+    tags:
+      - v*
```

Comment:
Hey, what do you think about still building in every run, but only publishing on a tag? That would make us more certain about the whole build process.